driftbase/api/events.py

```diff
@@ -29,6 +30,13 @@ def drift_init_extension(app, **kwargs):
     endpoints.init_app(app)
 
 
+default_eventlog_config = dict(eventlog=dict(max_batch_size=5, events_blacklist=[]))
```

Comment:
I think put `drift.*` in the defaults, or even add it explicitly to the config value at runtime, so that without config, things still work as before.

Reply:
Just added it into the default, but explicitly removing it would be even better